analysis_options.yaml

```diff
@@ -1,25 +1,38 @@
-include: package:lints/recommended.yaml
+# https://dart.dev/guides/language/analysis-options
+include: package:dart_flutter_team_lints/analysis_options.yaml
+
+analyzer:
+  language:
+    strict-casts: true
+  errors:
+    # For the generated file
+    lines_longer_than_80_chars: ignore
 
 linter:
   rules:
-    - always_declare_return_types
-    # - annotate_overrides
-    - avoid_empty_else
-    - avoid_init_to_null
-    # - avoid_return_types_on_setters
-    - camel_case_types
-    # - constant_identifier_names
-    - empty_constructor_bodies
-    - hash_and_equals
-    - library_names
-    - library_prefixes
-    # - non_constant_identifier_names
+    - avoid_bool_literals_in_conditional_expressions
```

Comment:
Can we not just drop all this custom stuff now?

Reply:
I'm using the bits from `string_scanner`. 🤷

Reply:
can you clarify?

Reply:
I dropped all the extra crazy. Just kept what was there (minus the dupes)